benchmarks/experiment_configs/datasets/multitext.json

```diff
@@ -0,0 +1,4 @@
+{
+  "dataset_name": "MultiText",
```

Comment:
Has this been modified to use 4 datasets instead of 5? I remember there being some issue with the reviews.

Reply:
see multitext-4updates.json